src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -57,7 +57,7 @@
     {%- endif -%}
     <select {{ block('widget_attributes') }}{% if multiple %} multiple="multiple"{% endif %}>
         {%- if empty_value is not none -%}
-            <option value=""{% if required and value is empty %} selected="selected"{% endif %}>{{ empty_value|trans({}, translation_domain) }}</option>
+            <option value=""{% if required and value is empty %} selected="selected"{% endif %}>{{ empty_value is empty ? empty_value : empty_value|trans({}, translation_domain) }}</option>
```

Comment:
also since we know that empty_value can only be a string at this point (null is also ruled out), we can just do `{{ empty_value != '' ? empty_value|trans({}, translation_domain) }}`